projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -214,6 +212,8 @@ public function is_supported_post() {
 	 * Enqueue block editor scripts and styles.
 	 */
 	public function enqueue_block_editor_scripts() {
+		global $publicize;
```

Comment:
Is it the use of the global you would like to avoid? We can call `publicize_init` to get the reference, but we do need to use the methods on that object to create the initial state.

Reply:
If we were hoping to get this PR in for 11.4-beta, could the approach used here be addressed in a future PR?

Reply:
@samiff Yes, that's what we decided :) p1664282377516619-slack-CBG1CP4EN